src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -196,10 +194,47 @@ public void deleteAttachment(String requestId, String companyNumber, String obje
                 () -> new AttachmentNotFoundException(String.format(ATTACHMENT_NOT_FOUND_MESSAGE, attachmentId))
         );
 
+        Map<String, Object> logMap = buildLogMap(companyNumber, objectionId);
+        deleteFromS3(requestId, attachmentId, logMap);
+
         attachments.remove(attachment);
 
         objection.setAttachments(attachments);
 
         objectionRepository.save(objection);
+
+    }
+
+    private void deleteFromS3(String requestId, String attachmentId, Map<String, Object>  logMap) throws ServiceException {
+        try {
+            FileTransferApiClientResponse response = fileTransferApiClient.delete(requestId, attachmentId);
+            if (response == null || response.getHttpStatus() == null) {
+                String message = String.format(DELETE_ERROR_MESSAGE_SHORT, attachmentId);
+                logger.infoContext(requestId, message, logMap);
```

Comment:
Rather than 'infoContext' this and log messages below should use 'errorContext'.

Reply:
Well we have used info context in other places the reason being is that error context is used when we have an exception to pass in.

Reply:
Ah ok. I thought there'd be an overloaded errorContext method that doesn't need an exception, but if there isn't then I guess infoContext is the one to use.